### PR TITLE
DBZ-7647 Adds capture.mode.full.update.type entry to config props table

### PR DESCRIPTION
**Pull request by roldanbob**
[DBZ-7647](https://issues.redhat.com/browse/DBZ-7647)

Updates the MongoDB documentation by adding an entry for `capture.mode.full.update.type` to  the table of Advanced connector configuration properties.

Tested in a local Antora build.

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/mongodb.adoc [hunk 1]
```diff
@@ -1813,6 +1815,20 @@ The following _advanced_ configuration properties have good defaults that will w
 |Default
 |Description
 
+|[[mongodb-property-capture-mode-full-update-type]]<<mongodb-property-capture-mode-full-update-type, `+capture.mode.full.update.type+`>>
+|`lookup`
+|Specifies how the connector looks up the full value of an updated document when the xref:mongodb-property-capture-mode[`capture.mode`] is set retrieve full documents.
+The connector retrieves full documents when `capture.mode` is set to one of the following options:
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/mongodb.adoc:1821]:
```suggestion
The connector retrieves full documents when its `capture.mode` is set to one of the following options:
```

**Comment by jpechane** [comment]
@jcechace Could you please proof-read the change? Feel free to merge PR yourself if it is OK for you.

**Comment by jcechace** [comment]
@roldanbob LGTM. Maybe adding an info that post-pre images need to be enable per collection would be a good idea? https://www.mongodb.com/docs/manual/changeStreams/#change-streams-with-document-pre--and-post-images

**Comment by roldanbob** [comment]
> @roldanbob Applied, thanks! Is this going to be backported to 2.5 branch?

@jpechane Yes, I think that's what we want, right? I realized that this was missing when I was looking at the 2.5 requirement to enable post-image support in MongoDB.

**Comment by jpechane** [comment]
Yes, we do. I just updated the fix version in Jira.

**Pull request merged**